spec/services/broadcasts/welcome_notification/generator_spec.rb

```diff
@@ -10,10 +18,44 @@
     end
 
     context "when sending a welcome_thread notification" do
-      xit "generates the appropriate broadcast to be sent to a user"
-      xit "it sends a welcome notification for that broadcast"
-      xit "it does not send duplicate welcome notification for that broadcast"
-      xit "does not send a notification to a user who has commented in a welcome thread"
+      let(:welcome_thread_article) { create(:article, title: "Welcome Thread") }
+      let(:welcome_thread_comment) { create(:comment, commentable: welcome_thread_article, user: user) }
+      let(:article) { create(:article) }
+      let(:comment) { create(:comment, commentable: article, user: receiving_user) }
+
+      it "generates the appropriate broadcast to be sent to a user", :aggregate_failures do
+        expect(welcome_broadcast).to be_kind_of(Broadcast)
+        expect(welcome_broadcast.type_of).to eq "Welcome"
+      end
+
+      it "sends a welcome notification for that broadcast" do
+        welcome_broadcast
+        described_class.new(receiver_id: receiving_user.id)
+        sidekiq_perform_enqueued_jobs do
```

Comment:
Hm, why do we need to use sidekiq_perform_enqueued_jobs here?

Reply:
I thought it was necessary for getting the Notifications... 🤔